loadgen/test_settings_internal.cc

```diff
@@ -388,10 +388,42 @@ int TestSettings::FromConfig(const std::string &path, const std::string &model,
   size_t val;
 
   // keys that apply to all scenarios
+  if (lookupkv(model, scenario, "mode", &val, nullptr)) {
+     switch (val) {
+       case 0 : mode = TestMode::SubmissionRun;
+                break;
+       case 1 : mode = TestMode::AccuracyOnly;
+                break;
+       case 2 : mode = TestMode::PerformanceOnly;
+                break;
+       case 3 : mode = TestMode::FindPeakPerformance;
+                break;
+       default : LogDetail([val = val](AsyncDetail &detail) {
```

Comment:
I think [val] does the same thing as [val = val].
Also, looks like val isn't used atm.